README.rst

```diff
@@ -51,6 +52,59 @@ Accepts an open-ended set of paired lists, each expected to provide an estimate
   19866960, 5437.757350231803
 
 
+Approximating means
+~~~~~~~~~~~~~~~~~~~
+
+Estimate a mean and approximate the margin of error. Uses a simulation approach. 
+First the number of units in each bin is simulated (assuming a normal distribution around the estimate). 
+Then for each unit a value within the bin is simulated (assuming a uniform distribution within each bin).
+We can then calculate the mean directly from the simulated data. We repeat the simulation many times to calculate an empirical margin of error.
+
+Expects a list of dictionaries that divide the full range of data values into continuous categories. Each dictionary should have four keys:
+
+.. list-table::
+  :header-rows: 1
+
+  * - key
+    - value
+  * - min
+    - The minimum value of the range
+  * - max
+    - The maximum value of the range
+  * - n
+    - The number of people, households or other units in the range
+  * - moe
+    - The margin of error for the number of units in the range
+
+Also expects the number of simulations to run (default = 50) and an optional seed if you want your results to be replicable. 
+Note that due to the simulation, this function takes some time to run. The example takes a little over a minute. 
+Increasing the number of runs should increase your precision (with diminishing returns) at the cost of a slower run time.
+If you do not set the seed, another call of the function with the same inputs is not guaranteed to return the same answer.
+
+.. code-block:: python
+
+ >>> income = [
+            dict(min=2499, max=9999, n=7942251, moe=17662),
```

Comment:
Some concept of what that false lowest or highest value is. If it's not a "jam value," what is it? A "jam input?" A "minimum extent?" Something else?

Reply:
This could be related to the ["midpoint estimator"](https://arxiv.org/pdf/1402.4061.pdf). If we had a midpoint estimate, we could obtain the upper bound. For the lower bound, I think we can just use zero.

Reply:
Would it make sense to make a helper function that estimates the midpoint and then returns the associated upper bound? Then the `approximate_mean` function doesn't have to worry about that part of the problem, the estimate will just be included as part of the input. 

Relatedly, we could implement a midpoint estimated version of the mean in addition to the simulation approach since that seems to be something other people use.

Reply:
I changed my mind about what to do here. It would be hard to recommend appropriate bounds for every potential table. Instead, I added an option to use the Pareto distribution instead of uniform for the last bin (useful for income which is probably a big use case). Users still need to choose upper and lower bounds, but the documentation now recommends assessing sensitivity of results based on these choices.